rpc/src/rpc.rs

```diff
@@ -3422,6 +3435,14 @@ pub mod rpc_full {
             meta: Self::Metadata,
             config: Option<RpcContextConfig>,
         ) -> Result<RpcResponse<u64>>;
+
+        #[rpc(meta, name = "getRecentPrioritizationFees")]
+        fn get_recent_prioritization_fees(
+            &self,
+            meta: Self::Metadata,
+            pubkey_strs: Vec<String>,
```

Comment:
Currently, this requires passing an empty vec to use this method with commitment, but without any pubkeys, eg:
```
curl -X POST -H "Content-Type: application/json" -d '{"jsonrpc": "2.0","id":1,"method":"getRecentPrioritizationFees","params":[[],{"commitment":"confirmed"}]}' 127.0.0.1:8899
```
Not sure how much to bend over backward to make both things optional.

Reply:
honestly it doesn't really bother me, at least for V1. We could change in future when user feedbacks are in

Reply:
Except we have a backward compatibility policy...
I guess if we can get feedback before this moves to the beta branch, we can do it that way.